beacon_node/client/src/builder.rs

```diff
@@ -433,6 +450,20 @@ where
 
                 debug!(context.log(), "Downloaded finalized block");
 
+                let blobs = if block.message().body().has_blobs() {
+                    debug!(context.log(), "Downloading finalized blobs");
+                    let blobs = remote
+                        .get_blobs::<TEthSpec>(BlockId::Root(block.canonical_root()), None)
+                        .await
+                        .map_err(|e| format!("Error fetching finalized blocks from remote: {e:?}"))?
```

Comment:
```suggestion
                        .map_err(|e| format!("Error fetching finalized blobs from remote: {e:?}"))?
```

Reply:
good catch, fixed in f20bd359e6c4ce4cf21b07cb55bf934740de83a2